tests/test_modeling.py

```
@@ -13,370 +13,325 @@
-            basic_setup['token_embeds'][1, 2]
-        )
-    
+        assert torch.allclose(words_embedding[1, 0], basic_setup["token_embeds"][1, 2])
+
     def test_creates_valid_mask(self, basic_setup):
         """Should create mask based on text_lengths."""
         words_embedding, mask = extract_word_embeddings(**basic_setup)
-        
+
         # First batch has text_length=2, so first 2 positions should be True
         assert mask[0, 0] == True
```

Ruff (0.15.1)

[error] Avoid equality comparisons to `True`; use `mask[0, 0]:` for truth checks

Replace with `mask[0, 0]`

(E712)